LipidMatch_Flow_3.5/Background_Files_LipidMatch_Flow/LipidMatch_Flow/python_scripts/EICgen/EIC_gen.py

```diff
@@ -0,0 +1,212 @@
+#!python3.8
+#
+# Copyright 2022 Innovative Omics
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
I slapped the license on there when I wrote it out of good habit, but as far as I am concerned it can come off now, and remain under the general license you use with your project. 

I generally don't put my name on code by habit because I am a code-hippy, but I am happy to put my name and email on it for purposes of helping the project stay organized, and keep up with bug fixes!